apps/build.gradle

```diff
@@ -121,6 +121,33 @@ subprojects {
         }
     }
 
+    //moved docker here because can't have two palantir in /build.gradle
+    //also means we don't need the dockerignore anymore
```

Comment:
should be a PR comment not a code comment

Reply:
ssr catalogue does not use palantir

Reply:
it does now

Reply:
it does now